tests/api/conftest.py

```diff
@@ -60,3 +62,13 @@ def client() -> Generator:
             get_k8s().crdapi = unittest.mock.Mock()
         with TestClient(app) as c:
             yield c
+
+
+@pytest.fixture()
+def mock_v3io_client(monkeypatch) -> Generator:
```

Comment:
Since this behave as a fixture and not really a function (i.e. you're not calling it, to use it you're just adding it as a arg to your test) I suggest to call it `v3io_client_mock`

Reply:
Will be iterated on in next PR, removing for now